routes/quiz.js

```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router({ mergeParams: true })
+
+const {
+  roles: { TUTOR }
+} = require('~/consts/auth')
+const quizController = require('~/controllers/quiz')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const { authMiddleware, restrictTo } = require('~/middlewares/auth')
```

Comment:
Move these imports to the top of the file.

Reply:
i think it's no need cause import from **node modules** should be separate from local imports